rpi5.yaml

```diff
@@ -0,0 +1,260 @@
+        - [DOM0_OS, "%{DOM0_OS}"]
+        - [PREFERRED_VERSION_xen,"4.19.0+git"]
+        - [PREFERRED_VERSION_xen-tools, "4.19.0+git"]
+        - [PREFERRED_VERSION_u-boot, "2024.04"]
+
+        # RPI specific
+        - [ENABLE_UART, "1"]
+        - [RPI_USE_U_BOOT, "1"]
+        - [PREFERRED_PROVIDER_u-boot-default-script, "xt-rpi-u-boot-scr"]
+        - [UART_BOORLOADER, "1"]
+        - [KERNEL_IMAGETYPE_UBOOT, "Image"]
+        - [DISTRO_FEATURES:append, " xen"]
+
+      build_target: "%{XT_DOMD_IMAGE}"
+      layers:
+        - "../../poky/meta"
+        - "../../poky/meta-poky"
+        - "../../poky/meta-yocto-bsp"
+        - "../../meta-virtualization"
+        - "../../meta-raspberrypi"
+        - "../../meta-selinux"
+        - "../../meta-openembedded/meta-oe"
+        - "../../meta-openembedded/meta-networking"
+        - "../../meta-openembedded/meta-perl"
+        - "../../meta-openembedded/meta-python"
+        - "../../meta-openembedded/meta-filesystems"
+        - "../../meta-xt-common/meta-xt-domx"
+        - "../../meta-xt-common/meta-xt-driver-domain"
+        - "../../meta-xt-rpi5"
+
+      target_images:
+        - "tmp/deploy/images/%{RPI_MACHINE}/xen-%{RPI_MACHINE}"
+        - "tmp/deploy/images/%{RPI_MACHINE}/xenpolicy-%{RPI_MACHINE}"
+        - "tmp/deploy/images/%{RPI_MACHINE}/%{XT_XEN_DTB_NAME}"
+        - "tmp/deploy/images/%{RPI_MACHINE}/bcm2712-rpi-5-b.dtb"
+        - "tmp/deploy/images/%{RPI_MACHINE}/%{XT_DOMD_IMAGE}-%{RPI_MACHINE}.rootfs.ext3"
+        - "tmp/deploy/images/%{RPI_MACHINE}/Image-%{RPI_MACHINE}.bin"
+        - "tmp/deploy/images/%{RPI_MACHINE}/bootfiles/bootcode.bin"
+        - "tmp/deploy/images/%{RPI_MACHINE}/bootfiles/config.txt"
+        - "tmp/deploy/images/%{RPI_MACHINE}/bootfiles/cmdline.txt"
+        - "tmp/deploy/images/%{RPI_MACHINE}/bootfiles/fixup.dat"
+        - "tmp/deploy/images/%{RPI_MACHINE}/bootfiles/fixup_cd.dat"
+        - "tmp/deploy/images/%{RPI_MACHINE}/bootfiles/fixup_db.dat"
+        - "tmp/deploy/images/%{RPI_MACHINE}/bootfiles/fixup_x.dat"
+        - "tmp/deploy/images/%{RPI_MACHINE}/bootfiles/fixup4.dat"
+        - "tmp/deploy/images/%{RPI_MACHINE}/bootfiles/fixup4cd.dat"
+        - "tmp/deploy/images/%{RPI_MACHINE}/bootfiles/fixup4db.dat"
+        - "tmp/deploy/images/%{RPI_MACHINE}/bootfiles/fixup4x.dat"
+        - "tmp/deploy/images/%{RPI_MACHINE}/bootfiles/start.elf"
+        - "tmp/deploy/images/%{RPI_MACHINE}/bootfiles/start_cd.elf"
+        - "tmp/deploy/images/%{RPI_MACHINE}/bootfiles/start_db.elf"
+        - "tmp/deploy/images/%{RPI_MACHINE}/bootfiles/start_x.elf"
+        - "tmp/deploy/images/%{RPI_MACHINE}/bootfiles/start4.elf"
+        - "tmp/deploy/images/%{RPI_MACHINE}/bootfiles/start4db.elf"
+        - "tmp/deploy/images/%{RPI_MACHINE}/bootfiles/start4cd.elf"
+        - "tmp/deploy/images/%{RPI_MACHINE}/bootfiles/start4x.elf"
+#      additional_deps:
+#        - "../%{ZEPHYR_DOM0_DIR}/%{ZEPHYR_DOM0_BUILD_DIR}/zephyr/zephyr.bin"
```

Comment:
Please remove this. This is not the real comment, but something like the development notes 'for later usage'.